examples/ecommerce-netlify-functions/dummyProducts.js

```diff
@@ -0,0 +1,20 @@
+const { Product } = require('./models');
+const mongoose = require('mongoose');
+
+async function createProducts() {
+    await mongoose.connect('mongodb://localhost:27017/netlify');
+
+    await Product.create({
+        productName: 'Dummy Product',
```

Comment:
Make this data a little better. Let's just take the product name, price, and add an `image` field. Take the data from here: https://github.com/temporalio/temporal-ecommerce/blob/5b1a2cd7e2b27472c92675ddec84365e9828930d/shared.go#L13-L41